packages/webui/src/client/ui/SegmentTimeline/SegmentTimelineContainer.tsx

```diff
@@ -706,6 +748,8 @@ const SegmentTimelineContainerContent = withResolvedSegment(
 							showCountdownToSegment={this.props.showCountdownToSegment}
 							fixedSegmentDuration={this.props.fixedSegmentDuration}
 							showDurationSourceLayers={this.props.showDurationSourceLayers}
+							onSegmentSelect={this.handleSegmentSelect}
+							isSelected={this.isSelected(this.props.segmentui._id)}
```

Comment:
See comment on `SegmentTimelineContainerWithSelection`. Also, if there's good reason for this setup of functions - `this.isSelected` knows the props, so this could easily be `this.isSelected()` - _BUT_, I feel like this is completely unneccesary and might as well just be `isSelected={this.props.isSelected}`, if `SegmentTimelineContainerWithSelection` is simplified